packages/experimental-app-router/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@faustwp/experimental-app-router",
+  "version": "0.0.0",
+  "description": "Experimental: A Faust package to support Next.js' App Router",
+  "main": "dist/cjs/index.js",
+  "module": "dist/mjs/index.js",
+  "types": "dist/cjs/index.d.ts",
+  "scripts": {
+    "dev": "concurrently \"npm:watch-*\" --prefix-colors \"auto\"",
+    "build": "concurrently \"npm:build-*\" --prefix-colors \"auto\"",
+    "prebuild": "npm run clean",
+    "postbuild": "npm run package",
+    "clean": "rimraf dist",
+    "format": "prettier --write .",
+    "package": "node ../../scripts/package.js",
+    "prepublish": "npm run build",
+    "test:coverage:ci": "jest --ci --json --coverage --testLocationInResults --outputFile=report.json",
+    "test:coverage": "jest --coverage",
+    "test:watch": "jest --watch",
+    "test": "jest",
+    "build-js": "tsc -p .",
+    "build-cjs": "tsc -p tsconfig-cjs.json",
+    "watch-js": "concurrently \"npm:build-js -- --watch\" \"npm:build-cjs -- --watch\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wpengine/faustjs.git"
+  },
+  "author": "Faust Team",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/wpengine/faustjs/issues"
+  },
+  "homepage": "https://github.com/wpengine/faustjs#readme",
+  "engines": {
+    "node": ">=16",
+    "npm": ">=8"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^5.17.0",
+    "@types/node": "^20.4.6",
+    "@types/testing-library__react": "^10.2.0",
```

Comment:
I get a notice about this not being needed. Do we need it for some reason?

> npm WARN deprecated @types/testing-library__react@10.2.0: This is a stub types definition. testing-library__react provides its own type definitions, so you do not need this installed.

Reply:
Nice catch, looks like that dep is no longer needed. I've removed it 👍